_meeting-notes/2019-10-29.md

```diff
@@ -0,0 +1,143 @@
+  2. Identify blockers / Answer Questions `20min`
+  3. This week's planning `20min`
+- **Reference:** 
+  - [Last week meeting minute](https://github.com/ipfs/testground/pull/69)
+  - [Trello Board](https://trello.com/b/O9z3ljaH/testground-ignition-%F0%9F%9A%80)
+  - [Testing Infra Design](https://github.com/ipfs/testground/blob/master/docs/SPEC.md)
+  - [Testing Inventory](https://github.com/ipfs/testground/blob/master/docs/test-inventory.md)
+  - [Testing Improvement Tracker](https://docs.google.com/spreadsheets/d/1xyqyGUF-oe3x9ln88YonVeOMWWdknik74lVgL_3dBY8/edit#gid=0)
+
+:::
+
+## :mega: Update (Done, Shipped, Made Progress)
+
+- ==@raulk==
+    - Preparing Testground Hack Week...
+    - Catching up with things across the board after returning from the oriental tour.
+    - Finishing up exec:go builder (builds a test plan into a go executable, Dockerless) and local:exec runner (runs an executable).
+- ==@daviddias==
+  - This last week was slow for me as notified in the [previous update (🦷 + ✈️)](https://github.com/ipfs/testground/blob/master/_meeting-notes/2019-10-22.md#mega-update-done-shipped-made-progress)
+  - My sidequest of getting the go-ipfs as a Library tutorial is now complete
+    - https://github.com/ipfs/go-ipfs/tree/master/docs/examples/go-ipfs-as-a-library
+    - PR - https://github.com/ipfs/go-ipfs/pull/6695
+    - Post - https://ipfs.io/blog/073-go-ipfs-as-a-library/
+  - Continue the work on the Plan 1 - Chew Large Datasets https://github.com/ipfs/testground/pull/58
+  - Continue the work on creating the Spec for the Test Plans
+    - Augmented the plans and moved the existing ones to their own folder https://github.com/ipfs/testground/pull/77
+    - Reviewed Steven's review https://github.com/ipfs/testground/pull/81
+- ==@jimpick==
+  - Discussed with Dirk about bitswap tests + AWS credentials
+    - https://github.com/ipfs/testground/pull/82
+  - Ansible!
+    - https://github.com/ipfs/testground/issues/84
+    - (spent the whole weekend learning it)
+    - Dynamic inventory for a cluster using EC2 tags
+    - Connectivity test
+    - Redis
+    - Filebeat
+    - Networking (GRE tunnels variant)
+- ==@stebalien==
+
+
+
+## :question: Questions (Blockers, Technical Design needs)
+> We only have 30 mins for this section, use your time wisely
+
+- (@dirkmc, 5 min)
+  - I have good knowledge of Bitswap codebase but not as much knowledge of Bitswap in the wild
+    I'd love to get some feedback on parameters for a realistic, real-world scenario
+    - Session Discovery (session queries connected peers and then asks DHT)
+    - How often do timeouts actually occur in practice once a peer has connected?
+    - Ephemeral nodes
+    - Request patterns: Wikipedia vs Large file vs Small files vs ...
+  - I think it makes sense for me to start helping out with testground (beyond just writing tests 🐹)
+    - Is testground at a point where that makes sense?
+    - Is there a good chunk for me to break off and work on?
```

Comment:
```suggestion
    - Is there a good chunk for me to break off and work on?
      - A: No, it would be better if you interfaced with the team as a user and builder of one of the plans. (I'm sure you will find bugs along the way and provide patches)
```